content/influxdb3/clustered/install/set-up-cluster/prerequisites.md

```diff
@@ -189,12 +189,12 @@ update an InfluxDB cluster.
 > separately. The Helm chart installs the kubit operator.
 
 Use `kubectl` to install the [kubecfg kubit](https://github.com/kubecfg/kubit)
-operator **v0.0.18 or later**.
+operator **v0.0.22 or later**.
 
 <!-- pytest.mark.skip -->
 
 ```bash
-kubectl apply -k 'https://github.com/kubecfg/kubit//kustomize/global?ref=v0.0.19'
+kubectl apply -k 'https://github.com/kubecfg/kubit//kustomize/global?ref=v0.0.22'
```

Comment:
I get a 404 for this URL, but maybe I don't have permissions?
If the URL isn't valid, should it be `https://raw.githubusercontent.com/kubecfg/kubit/refs/tags/v0.0.22/kustomize/global/kustomization.yaml`?

Reply:
No worries, thanks for the review!